pkg/cmd/cli/describe/deployments.go

```diff
@@ -105,6 +105,9 @@ func (d *DeploymentConfigDescriber) Describe(namespace, name string, settings kc
 				deployment = &item
 			}
 		}
+		if deployment == nil {
```

Comment:
Can you open an issue so we can track that change? It's a problem that we haven't tracked it so far.